staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/values.go

```diff
@@ -554,7 +560,7 @@ func (t *unstructuredMap) Find(key ref.Val) (ref.Val, bool) {
 		}
 	}
 	if v, ok := t.value[k]; ok {
-		if v == nil {
+		if isObject && v == nil {
```

Comment:
took me a minute to figure out what was going on here... might be worth a comment